src/app/components/ContactViewProperty.js

```diff
@@ -61,9 +61,9 @@ const ContactViewProperty = ({
             return <a href={`tel:${value}`}>{value}</a>;
         }
         if (['bday', 'anniversary'].includes(field)) {
-            const date = moment(value);
-            if (date.isValid()) {
-                return date.format('LL');
+            const [date] = [parseISO(value), toDate(value)].filter(isValid);
+            if (date) {
+                return format(date, 'PP');
```

Comment:
```js
import { dateLocale } from 'proton-shared/lib/i18n';
return format(date, 'PP', { locale: dateLocale });
```

Reply:
Yep, forgot about that. Gonna add it in the `ContactKeysTable` also